gulp/tasks/cacheBust.js

```diff
@@ -0,0 +1,15 @@
+import gulp from 'gulp';
+import rev from 'gulp-rev-append';
+import {settings} from '../config';
+import {isProdBuild} from '../commandLineArgs';
+
+function cacheBust(done) {
+	if (isProdBuild()) {
+		return gulp.src(`${settings.destinations.prod.markup}/**/*.html`)
```

Comment:
I wondered a minute. Then I got that it needs to be applied to the destination because it doesn’t work with:

```
<!-- build:css assets/css/index.min.css?rev=@@hash -->
```

:grinning: